src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -65,6 +63,9 @@ public sealed class LegacyPackageReferenceProject : PackageReferenceProject
             InternalMetadata.Add(NuGetProjectMetadataKeys.ProjectId, projectId);
 
             ProjectServices = projectServices;
+
+            InstalledPackages = new(); // This colleciton is a Dictionary whose key type is packageId
+            TransitivePackages = new(); // This colleciton is a Dictionary whose key type is packageId
```

Comment:
In our coding guidelines, we say only use `var` where the type is obvious. In the same spirit, I don't think that using `new()` is appropriate, since neither `InstalledPackages` or `TransitivePackages` types are obvious here. In fact, neither appear to be defined in this file. I'm guessing it's defined in the parent type, so I think that the parent's constructor should do this, so that the initialization happens in the same class that "owns" the property/field.

Reply:
Fixed.!